slsDetectorSoftware/src/Detector.cpp

```diff
@@ -250,14 +250,14 @@ void Detector::setTimingMode(defs::timingMode value, Positions pos) {
 void Detector::acquire() { pimpl->acquire(); }
 
 void Detector::startAcquisition() {
-    if (getUseReceiverFlag({}).squash())
+    if (getUseReceiverFlag({}).squash(true))
         pimpl->Parallel(&slsDetector::startReceiver, {});
     pimpl->Parallel(&slsDetector::startAcquisition, {});
 }
 
 void Detector::stopAcquisition() {
     pimpl->Parallel(&slsDetector::stopAcquisition, {});
-    if (getUseReceiverFlag({}).squash()) // TODO: problem for acquire()
+    if (getUseReceiverFlag({}).squash(true)) 
```

Comment:
Same here...

Reply:
I removed it for the  getUseReceiverFlag. But I need them for the Parallel call after stopAcquisition. To remove it from there, I need to edit multiSlsDetector::Parallel to allow default {} for positions. Shall I do that? Or was it omitted for some reason?